docs/attestations.md

```diff
@@ -0,0 +1,56 @@
+# Attestations
+
+> [!CAUTION]
+> This feature is still experimental.
+
+## Introduction
+
+BCR now accepts build attestations in order to increase security for all BCR users.
+This feature is still experimental and thus optional, but eventually we'll reject all PRs without proper
```

Comment:
> but eventually we'll reject all PRs without proper attestation.

This statement is a bit scary. For some projects like "zlib", the original project owner probably doesn't care about Bazel build so it's still unclear if it would be possible to generate attestations for those modules. Can we ease the tone a bit? Maybe like "eventually we expect all modules that natively build with Bazel to also generate build attestations"